WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -93,14 +102,21 @@ public void onSaveInstanceState(Bundle outState) {
         outState.putSerializable(KEY_USERNAME_RESULTS, mUsernameResults);
         outState.putString(KEY_ROLE, mRole);
         outState.putString(KEY_CUSTOM_MESSAGE, mCustomMessage);
+        outState.putBoolean(KEY_INVITE_IN_PROGRESS, mInviteOperationInProgress);
```

Comment:
With `setRetainInstance(true)`, saving the `mInviteOperationInProgress` boolean is actually not needed as the Fragment instance is kept. Basically, same holds true for all these instance variables anyway. May I suggest removing this manual saving? Thanks!